.github/workflows/build-and-test.yml

```diff
@@ -8,6 +8,7 @@ on:
     paths:
       - '.github/workflows/build-and-test.yml'
       - '.github/workflows/scripts/**'
+      - 'agent-bundle/**'
```

Comment:
In order to have the relocated agent code externally available I'm thinking it should go in `pkg/signalfx-agent`. Would having the bundle in `pkg/signalfx-agent/bundle` make sense?

Reply:
Makes sense to me.